### DISCUSSION TITLE
Need an easier way to brand this without forking.

### DISCUSSION THREAD

**Discussion by cbutler-captech** [microsoft/sample-app-aoai-chatGPT#438] [General]
I think this repo would be much more useful if there was an easy way to brand it with environment variables vs forking.  For example, HEADER_LOGO_URL=https://my.company.com/logo

Here are the 3 things that I am being asked to do, that I am sure every company wants:

- Change the Logo
- Change the header title to the right of the logo
- Change the main icon in the new chat window
- Add a small paragraph under the new chat icon that has links to company policies

These things are simple and unfortunately most companies would default to a private fork to avoid having any private info (internal links, policy info).  This means less time to contribute to the repo.  My team can potentially tackle some of these if approved and push them back into this repo since it can then be a public fork that funnels changes back.

**Comment by iMicknl** [comment]
See https://github.com/microsoft/sample-app-aoai-chatGPT/pull/563 for a proposal of this feature.
